### PR TITLE
Self review

### PR DESCRIPTION
**Pull request by mnaibei**
Added index.html
Added index.js

### CODE REVIEW

**Review by mnaibei — COMMENTED** [review]
#2 
@mnaibei 
Please make sure to add linters to your repo so as to ensure your code conforms to best practices.
Please ensure your HTML layout conforms to HTML best practices

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @mnaibei  :wave: 

Your project page is looking fantastic so far :

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:



Congrats on your work so far! :clap:  

## Required Changes ♻️
- You are required to follow he guidelines from the lesson [How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)
    - In the lesson you are required to use inline comments. Use the video below to see how to use this. [video with an example](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d).
    - Have at least two comments. The comments you have are both just one comment. Kindly ensure you have 2 different comments
    - Also ensure that you give a clear explanation for requesting the changes.

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+          The man who saved a billion lives
+      </p>
+    <div id="img-div">
+      <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"> 
+    
+    <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+    </div>
+      
+     <p>If you have time, you should read more about this incredible human being on his <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug"  target="_blank"> Wikipedia entry.</a></p>
+
```
Comment (mnaibei) [inline index.html:33]: Please make sure to add linters to your repo so as to ensure your code conforms to best practices.

**Review by mnaibei — COMMENTED** [review]
Please add a professional README document detailing what your project is about.

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @mnaibei, 👋 👋  

Your project is complete! Great job for following the project requirement🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Has a proper Pull Request. ✔️
- No Linter errors ✔️
- Good use of inline comment ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**